README.md

```diff
@@ -1,2 +1,10 @@
 # sqlflowserver
 The gRPC proxy server of SQL engines
+
+## Build and Test
+
+According to https://bit.ly/2BCxGwH, we are supposed to commit the protoc-generated Go source files.  So, to build and test, we can run the following command:
```

Comment:
This PR doesn't contain generated `sqlflow.pb.go`, you mean "we are NOT supposed to commit ..."?

Reply:
Yes, we should git commit generated source files.

Go's convention is to git commit all auto-generated files. This is because Go's toolchain is very simple and Go Authors had a discussion and decided not to complicate the toolchain.

This works with the Go source file generated by goyacc, as well as *.pb.go files generated by protoc.  For example, protobuf's Go example has the generated file commited: https://github.com/grpc/grpc-go/blob/master/examples/helloworld/helloworld/helloworld.pb.go

Reply:
Sure, I will make this PR include the generated file.